pkg/reaper/README.md

```diff
@@ -121,6 +121,11 @@ Flags:
       --kubectl string                     Absolute path to the kubectl binary (default "/usr/local/bin/kubectl")
       --local-mode                         Use cluster external auth
       --max-kill-nodes int                 Kill up to N nodes per job run, considering throttle wait times (default 3)
+      --control-plane-node-count           Expected number of control plane nodes (default 3)
+      --node-healthcheck-interval          Time (in seconds) to wait between node healthchecks (default 5)
+      --node-healthcheck-timeout           Time (in seconds) to wait before node healthcheck timeout (default 1800)
+      --cluster-id                         Unique cluster identifier; used for node reaper locks. Only required if --locks-table-name is set. 
+      --locks-table-name                   DynamoDB table name for storing the lock objects (optional)
```

Comment:
Missing default value

Reply:
That's on purpose, since it's optional.

Reply:
But it has a default value, it should be mentioned here imo

Reply:
Everything that has a default value is optional

Reply:
oh the actual default shoudl be `""`, I'll fix.